packages/genesys-spark-components/src/components/beta/gux-tooltip-beta/gux-tooltip.tsx

```diff
@@ -110,7 +108,11 @@ export class GuxTooltip {
           tooltipId={this.id}
           ref={el => (this.baseTooltip = el)}
         >
-          <slot />
+          <span slot="content">
+            <slot name="content">
+              <slot />
```

Comment:
@thomas-c-dillon not sure if falling back to a default slot is a good idea here or not. Will discuss at standup.

Reply:
This makes sense syntactically. I can see where it is confusing though. Alternatively, as this is the beta version, we could just choose not to support the default slot here but that will require a larger version bump when we replace the stable version.

Alternatively, alternatively, we could make another component specifically for wrapping text node children that applies typography styles including word break (the sibling of gux-truncate).

Reply:
> we could just choose not to support the default slot here but that will require a larger version bump when we replace the stable version.

I suggest we leave it like I have it for now and depending on when we migrate the beta version to stable we can leave or change the api. before v5 leave the api in v5 remove the default slot.